pandas/tests/series/methods/test_sort_values.py

```diff
@@ -81,7 +82,8 @@ def test_sort_values(self, datetime_series):
             "you must create a copy"
         )
         with pytest.raises(ValueError, match=msg):
-            s.sort_values(inplace=True)
+            return_value = s.sort_values(inplace=True)
```

Comment:
this likely doesn't execute, can you revert (and if you are testing withing other raises tests)